roles/swift-ring/tasks/main.yml

```diff
@@ -13,3 +13,4 @@
   command: sudo -u swiftops /usr/local/bin/swifttool -i /home/swiftops/.ssh/id_rsa
            bootstrap --config /etc/swift/ring_definition.yml --outdir /etc/swift
   when: not should_boostrap|success
+  notify: restart swift services
```

Comment:
As for ordering, and " Is there a reason you don't like restarting services when rings are deployed?", it's more that I don't like starting the service when we know that the service won't start (because the rings are missing), so starting them just to have them crash and needing to restart them later feels... wrong. Also I still don't understand why this would only happen in the source install method and not in the package method.